src/clj/rems/service/test_data.clj

```diff
@@ -1,5 +1,7 @@
-(ns rems.db.test-data
-  "Populating the database with nice test data."
+(ns rems.service.test-data
+  "Generate and populate database with usable test data. Contains multiple high-level
+   test data helper functions. Separate functions are provided to generate complete
+   test and demo data, which create the same data, but with different users."
```

Comment:
judging both `(create-test-data!)` and `(create-demo-data!)` the user data seemed like key difference to me, and maybe good to point out?